types/opapp/objects/VideoBroadcastObject.d.ts

```diff
@@ -2,5 +2,13 @@ declare namespace OpApp {
     // @ts-ignore
     export interface VideoBroadcastObject extends OIPF.VideoBroadcastObject {
         onChannelChangeSucceeded: ( channel: OIPF.Channel, viewerChannel: OIPF.Channel, quiet: number ) => void;
+        readonly playState: playState;
```

Comment:
With the exception of playStateError property, these all exist on the VBO in the DAE Vol 5 R3 spec, so suggest most of these are moved to the OIPF.VideoBroadcastObject class